src/Presheaf/Hom.agda

```diff
@@ -13,7 +13,7 @@ open C
   { Run = λ{ a1 -> record { Obj = a1 ~> a2 ; _~_ = _≈_ ; ι = ι₂ ; ÷_ = ÷₂_ ; _⨾_ = _⨾₂_ } }
   ; Map = λ f → record { run = f ⨾_ ; map = f ^⨾′_ }
   ; Map′ = λ e → record { component = e ⨾′^_ }
-  ; Map-ι = record { component = λ{ a -> ι⨾ } }
+  ; Map-ι = record { component = λ{ _ -> ÷₂ ι⨾ } }
```

Comment:
I think needing ÷₂ is an indication that we messed up the direction of the law somewhere, no?

Reply:
So the tension here is between map-id and lunit.   The former has a canonical direction (relative to map-compose) by analogy to lax monoidal functors and lax natural transformations.   So if we messed up anything it would be lunit.  AFAIK there isn't a canonical direction for lunit, so I'm guessing Nate just chose the direction of reduction.

Reply:
Since this project is relatively self-contained, I think it makes sense to try to minimize the number of places where we need `÷₂`.  So I'm inclined to either swap the direction of `lunit` or to swap the direction of `map-compose` for consistency with `map-id` (whichever globally minimizes the instances of `÷₂`).